javatests/arcs/showcase/typevariables/Consumer.kt

```diff
@@ -11,7 +11,6 @@ class Consumer : AbstractConsumer() {
             .also { assertThat(it).hasSize(3) }
             .forEach {
                 assertThat(it.sku).endsWith("*****")
-                assertThat(it.name).isNotEmpty()
```

Comment:
Why is this check removed?

Reply:
Great question! The answer may involve a new thing to demonstrate in the showcase.
Name is not exposed on the handle, because it wasn't explicitly listed in the type variable constraint. However, the data can be fed to the consumer particle, but it's only accessible via reflection ("reflective" particles). 
Should I change the example to demonstrate this reflective capability?

Reply:
May be leave this example as is. It might be worthwhile to add another showcase example with the reflection capability.

Reply:
Cool, I'm happy with that approach.